library/src/main/java/com/novoda/simplechromecustomtabs/SimpleChromeCustomTabs.java

```diff
@@ -20,42 +19,35 @@
 
 public final class SimpleChromeCustomTabs implements WebNavigator, Connection, AvailableAppProvider {
 
-    private static Context applicationContext;
     private Connection connection;
     private WebNavigator webNavigator;
     private AvailableAppProvider availableAppProvider;
 
-    private SimpleChromeCustomTabs() {
-        //no-op
-    }
-
-    private static class LazyHolder {
-        private static final SimpleChromeCustomTabs INSTANCE = new SimpleChromeCustomTabs();
+    /**
+     * @deprecated There is no need to call it anymore in order to use this library,
```

Comment:
Interesting. I hadn't thought of it like that. I honestly spent way too long trying to come up with reasons for keeping the deprecated and tbh I give up 😆 

Keep it: We aren't misleading in the library, we don't need this anymore. We might run into issues later if we introduce it again and annoy a bunch of people using the library.
Remove it: We cover ourselves because it might be used at some point in the future.